packages/tools/kadena-cli/src/devnet/utils/simulation/coin/simulate.ts

```diff
@@ -0,0 +1,284 @@
+import type { ChainId } from '@kadena/client';
+import type { IAccount } from '../../../../constants/devnets.js';
+import {
+  defaultAccount,
+  simulationDefaults,
+} from '../../../../constants/devnets.js';
+import type { TransferType } from '../file.js';
+import { appendToLogFile, createFile } from '../file.js';
+import {
+  generateAccount,
+  getAccountBalance,
+  getRandomNumber,
+  getRandomOption,
+  isEqualChainAccounts,
+  seedRandom,
+  stringifyProperty,
+} from '../helper.js';
+import { crossChainTransfer } from './crosschain-transfer.js';
+import { safeTransfer } from './safe-transfer.js';
+import { transfer } from './transfer.js';
+
+const simulationTransferOptions: TransferType[] = [
+  'cross-chain-transfer',
+  'transfer',
+  'safe-transfer',
+];
+
+export const MARMALADE_TEMPLATE_FOLDER = 'src/devnet/contracts/marmalade-v2';
```

Comment:
This const isn't being used right? please remove it. (also it wouldn't work if used)

Reply:
Removed. Well spotted 👍🏽